Iguana2024/devops-2024
# Igd 22 adjust structure and ssh key authorization

## What type of PR is this? (check all applicable)


## Description
Refactored logic of adding ssh-keys to user vagrant through creating seperate directory for vagrant_user keys named machine_keys where you need to place both keys for machines to communicate with each other. And upgraded python script so every machine writes it's own unique reports.

## QA Instructions, Screenshots, Recordings
Login to your vagrant machine using you ssh_key and then check for reports in /vagrant directory

## [optional] Are there any post deployment tasks we need to perform?
